docs/guides/vm-architectures/README.md

```diff
@@ -0,0 +1,399 @@
+* **[Logical Layers](#logical-layers) (Logical Resilience).** This is not a physical resilience level but an architectural layer on top of any setup. It addresses the risk of data access conflicts by providing strong logical isolation between different teams or customers.
+
+### The decision tree
+
+<p align="center">
+<img src="decision-tree.webp" alt="Decision Tree" width="80%">
+</p>
+
+## Basic
+
+**For whom**: pet projects, developments/test stages, and non-critical systems monitoring.
+
+Installation guide reference: [https://docs.victoriametrics.com/guides/k8s-monitoring-via-vm-single](https://docs.victoriametrics.com/guides/k8s-monitoring-via-vm-single)
+
+**Key characteristics**: Single instance that does everything: stores, retrieves, and provides metrics.
+
+**Pros**:
+
+* **Straightforward.** Quick deployment without additional components  
+* **Cost-efficient.** It avoids redundant work, such as writing or transmitting the same data twice, thereby reducing both computational and network expenses. Additionally, there are no extra copies of data.
+
+**Cons**:
+
+* **Single point of failure.** No fault tolerance and no availability
+
+**Schema:**
+
+<p align="center">
+<img src="basic-architecture.webp" alt="Basic Architecture" width="40%">
+</p>
+
+### Unavailability Scenarios
+
+In this simplest setup, any single-node failure leads to temporary data unavailability or loss until the instance restarts or storage is restored. There are no built-in redundancy or replication layers.
+
+For this section, you can increase availability by utilizing backup and restore mechanisms on various levels: hardware, virtualization, persistence volume management, or application. VictoriaMetrics provides the [backup tools](https://docs.victoriametrics.com/victoriametrics/vmbackup/) to achieve that. 
+
+## Single Availability Zone
+
+**Target audience**: Single availability zone hosted system of any scale
+
+Installation guide reference: [https://docs.victoriametrics.com/guides/k8s-monitoring-via-vm-cluster](https://docs.victoriametrics.com/guides/k8s-monitoring-via-vm-cluster)
+
+High availability implementation: [https://docs.victoriametrics.com/guides/k8s-ha-monitoring-via-vm-cluster](https://docs.victoriametrics.com/guides/k8s-ha-monitoring-via-vm-cluster)
+
+**Key characteristics:** This is a complete VictoriaMetrics cluster, commonly running in a single Kubernetes cluster. Each component of the cluster: vminsert, vmselect, and vmstorage has multiple copies (replicas). The data is also copied and sharded between vmstorage nodes using the `--replicationFactor` setting on vminsert. [See the official documentation](https://docs.victoriametrics.com/victoriametrics/cluster-victoriametrics/#replication-and-data-safety) to determine the optimal replication factor for your needs.
```

Comment:
```suggestion
**Key characteristics:** This is a full VictoriaMetrics cluster, commonly running in a single Kubernetes cluster. Each component of the cluster: vminsert, vmselect, and vmstorage has multiple copies (replicas). The data is also copied and sharded between vmstorage nodes using the `--replicationFactor` setting on vminsert. [See the official documentation](https://docs.victoriametrics.com/victoriametrics/cluster-victoriametrics/#replication-and-data-safety) to determine the optimal replication factor for your needs.
```
Not sure if its "copied and sharded" - we replicate it, but don't shard it, don't we?